config/routes.rb

```diff
@@ -27,6 +27,7 @@
 
   post "/cart/:item_id", to: "cart#add_item"
   get "/cart", to: "cart#show"
+  patch "/cart/:item_id", to: "cart#increment_decrement"
```

Comment:
maybe we should make sure with Meg/Mike to make sure there's ok with this kind of route